### PR TITLE
refactor: using abi instead of json

### PR DESCRIPTION
**Pull request by JoowonYun**

### CODE REVIEW

File: x/executionlayer/client/cli/tx.go [hunk 1]
```diff
@@ -76,13 +77,20 @@ func GetCmdContractRun(cdc *codec.Codec) *cobra.Command {
 				return err
 			}
 
+			var jsonData []map[string]interface{}
+			err = json.Unmarshal([]byte(args[2]), &jsonData)
```
[the remark this answers is not in the document]
Comment (psy2848048) [reply x/executionlayer/client/cli/tx.go:81]: Works of me

```go
var prettyJson []byte
var jsonData []map[string]interface{}
if len(args[2]) > 0 {
	err = json.Unmarshal([]byte(args[2]), &jsonData)
	if err != nil {
		return err
	}
	prettyJson, err = json.Marshal(jsonData)
} else {
	prettyJson = []byte("")
}
```

**Comment by psy2848048** [comment]
> It runs OK https://github.com/hdac-io/swap/blob/master/docs/Methods.md#1-insert-an-allowance-cap-of-low-level-verification-in-kyc
> But it fails with code 5 (`casperlabs_contract::storage::read()` returned an error.)
> https://github.com/hdac-io/swap/blob/master/docs/Methods.md#2-insert-snapshot-record
> 
> Could you please take a look?

FYI, no problem in JSON parsing of gRPC util

File: x/executionlayer/client/cli/tx.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (JoowonYun) [reply x/executionlayer/client/cli/tx.go]: https://github.com/hdac-io/friday/pull/154/commits/4b2fb353a3a3061816c78221ac175a28193b14e6

**Comment by JoowonYun** [comment]
> > It runs OK https://github.com/hdac-io/swap/blob/master/docs/Methods.md#1-insert-an-allowance-cap-of-low-level-verification-in-kyc
> > But it fails with code 5 (`casperlabs_contract::storage::read()` returned an error.)
> > https://github.com/hdac-io/swap/blob/master/docs/Methods.md#2-insert-snapshot-record
> > Could you please take a look?
> 
> FYI, no problem in JSON parsing of gRPC util

Fixed in this https://github.com/hdac-io/friday/pull/154/commits/d0698bb501fc931f671459e715cd68c357b93268

It has problem is related to `deployhash`. 

And It's pending on this https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/29

**Pull request merged**